eng/tools/spec-gen-sdk-runner/src/utils.ts

```diff
@@ -294,7 +294,7 @@ export function searchRelatedTypeSpecProjectBySharedLibrary(
       try {
         const peerFiles = fs.readdirSync(path.resolve(options.specRepoFolder, peerPath));
         if (peerFiles.some((file) => options.searchFileRegex.test(file.toLowerCase()))) {
-          result[peerPath] = sharedLibraries;
+          result[normalizePath(peerPath)] = sharedLibraries;
```

Comment:
Can you call `path.normalize` directly?

Reply:
The typespecProjectResult alway get the key like "specification/contosowidgetmanager/Contoso.WidgetManager.Shared/main.tsp" in Windows and Linux and will do check in [code](https://github.com/Azure/azure-rest-api-specs/blob/c3a83e4696975664c8e0a660d900a0580a40f996/eng/tools/spec-gen-sdk-runner/src/change-files.ts#L64). The `path.normalize` will get the path with `//` in windows, so it will convert manually.